index.html

```diff
@@ -13,21 +13,27 @@
     <div class="container">
         <div class="section-bg-image">
         <header class="toolbar">
-            <nav class="nav-menu">
-                <a href="index.html" class="logo primary">Bornittah</a>
-                <ul class="menu">
-                    <li>
-                        <a href="#">Portifolio</a>
-                    </li>
-                    <li>
-                        <a href="#about">About</a>
-                    </li>
-                    <li>
-                        <a href="#contact">Contact</a>
-                    </li>
-                </ul>
-                <img src="./assets/icons/bars-solid.svg" class="navbar icon" alt="navbar-icon">
-            </nav>
+            <div class="overlay">
+                <nav class="nav-menu">
+                    <a href="index.html" class="logo primary">Bornittah</a>
+                    <ul class="menu">
+                        <li>
+                            <a  class="menu-link" href="#portfolio">Portifolio</a>
+                        </li>
+                        <li>
+                            <a  class="menu-link" href="#about">About</a>
+                        </li>
+                        <li> 
+                            <a  class="menu-link" href="#contact">Contact</a>
+                        </li>
+                    </ul>
```

Comment:
### OPTIONAL 

- [ ] It would be better for the desktop nav to start at 768px, as this will make your design look better at smaller screen sizes 👍🏿 
![Screenshot_202](https://user-images.githubusercontent.com/69638013/169042719-11656bf8-5e85-41b9-a411-a3f355085e6a.png)